lib/route/user.js

```diff
@@ -349,12 +349,10 @@ exports.postMeResendConfirmation = function(request, response){
 }
 
 exports.postMeLogout = function(request, response) {
+    response.clearCookie('connect.sid');
     if ('session' in request) {
         request.session.destroy();
-        return response.sendStatus(204);
+        delete request.session;
```

Comment:
request.session.destroy() is a wrapper for delete request.session, so you won't need line 355. I do agree that a 204 is necessary in both cases though.